scalaz-test/src/main/scala/ztest/Suite.scala

```diff
@@ -0,0 +1,36 @@
+package scalaz.test
+
+import java.lang.String
+import scala.Boolean
```

Comment:
I believe both of these are available in the `scalaz` object if you did `package scalaz; package test`

Reply:
Yes, please.